Externals.cfg

```
@@ -21,7 +21,7 @@ protocol = git
 repo_url = https://github.com/NOAA-EMC/UPP
 # Specify either a branch name or a hash but not both.
 #branch = develop
-hash = 22cfb88
+hash = dccb321
```

[`dccb321`](https://github.com/NOAA-EMC/UPP/commit/dccb32176930676ef2a258eb65571ab4e3f7e7a4) is an old hash from early april, the latest is [`1edfb3c`](https://github.com/NOAA-EMC/UPP/commit/1edfb3c71894ed0740f75692bd362a014b9c66aa); is there a reason not to update here?

```suggestion
hash = 1edfb3c
```

---

During the preparation for the SRW v2.1.0 release, I had to drop the version of UPP to mirror the version that the WM used.  I was told that this was to allow for the inline-post to work properly.

The WM at hash [e403bb4](https://github.com/ufs-community/ufs-weather-model/commit/e403bb43ce5f1e0b5a10a37d136cc84a205da7ce) uses the UPP at [dccb321](https://github.com/NOAA-EMC/UPP/commit/dccb32176930676ef2a258eb65571ab4e3f7e7a4).

I don't know if there is any truth to this, but that's the way that I've been updating the WM and UPP hashes since the v2.1.0 release.  I bring the WM hash to the HEAD of develop, then update UPP to the version used by the WM.  If it is safe to use the current HEAD of both repositories, then I don't mind updating them (though I will likely be asked to drop the UPP version in the release branch).

---

I don't think there's any reason the UPP hashes have to match, but I guess an abundance of caution is called for this close to a release. I'm okay with an older hash with that reason given.

---

Wen usually opens PRs in the WM to bring in the latest UPP changes on a monthly cadence.  Wen has been out for the last three weeks, which is why the UPP in the WM is so far behind.  I will likely update the WM one more time before the release, which should bring in a much more recent version of UPP.